drivers/clock_control/clock_control_mcux_mcg.c

```diff
@@ -37,9 +37,11 @@ static int mcux_mcg_get_rate(const struct device *dev,
 	clock_name_t clock_name;
 
 	switch ((uint32_t) sub_system) {
+#ifdef kCLOCK_McgFixedFreqClk
```

Comment:
Why is this needed?

Reply:
Because KW40W/KW41Z don't support it so if you build mcg on those platforms it fails to build.